cms/db/contest.py

```diff
@@ -79,7 +79,7 @@ class Contest(Base):
     languages = Column(
         ARRAY(String),
         nullable=False,
-        default=["C11 / gcc", "C++17 / g++", "Pascal / fpc"])
```

Comment:
I'm thinking this technically requires a schema updater, but to be fair we didn't write one for #1232 either, so... 🤷‍♂️